nuxeo/values.yaml

```diff
@@ -39,18 +39,29 @@ service:
 ## List of extra `containerPort`s for the nuxeo container.
 # extraContainerPorts:
 probePath: /nuxeo/runningstatus
+# Nuxeo pods will be restarted if probe fails during 1 minute
```

Comment:
I don't understand the math, 
will be helpful to provide the formula instead (final value can be desync)

Reply:
ok so up to periodSeconds * failureThreshold (5 * 12)

Reply:
Yes exactly, I just added the comment to quickly know what the probe is used for.
For the values, they are the kubernetes ones and so it is documented on their site.